### PR TITLE
fix: update a `QueryBuilder.returning` call because of the lucid breaking change

### PR DESCRIPTION
**Pull request by Julien-R44**
As mentioned here: https://github.com/adonisjs/auth/issues/195, the breaking change of lucid/knex about `returning` method broke the Auth package, so I fixed it
Migrated to the new version of Japa too

### CODE REVIEW

**Comment by thetutlage** [comment]
Let's keep PR's focused and small from next time. Makes it easy to revert if we have to :)

File: test-helpers/index.ts [hunk 1]
```diff
@@ -100,13 +100,13 @@ export async function setupApplication(
 				primary: {
 					client: 'sqlite3',
 					connection: {
-						filename: '${join(fs.basePath, 'primary.sqlite3')}',
+						filename: '${join(fs.basePath, 'primary.sqlite3').split(sep).join(posix.sep)}',
```
Comment (thetutlage) [inline test-helpers/index.ts:103]: A little confused on why we doing this?

**Comment by thetutlage** [comment]
Thanks for the PR :)

File: test-helpers/index.ts (hunk 1 above)
Comment (Julien-R44) [reply test-helpers/index.ts:103]: If we don't convert the path to POSIX format on windows I have a config/database.ts file that looks like this after running `setupApplication` :
```ts
  connections: {
    primary: {
      client: 'sqlite3',
      connection: {
        filename: 'C:\Users\Julien\\adonis\adonis-auth\test-helpers\__app\primary.sqlite3',
      },
    },
  }
````

So it doesn't work coz some characters are escaped and therefore impossible to launch the tests

**Pull request merged**